tests/system/general/test_45_FilterSolutions/test_distributed_filter/test_distributed_topology.py

```diff
@@ -0,0 +1,541 @@
+
+        with pytest.raises(RuntimeError) as info:
+            design.topology.center_z0_impedance = "55"
+        assert info.value.args[0] == "The Center Z0 property is not supported for the selected topology"
+        design.attributes.filter_class = FilterClass.BAND_PASS
+        design.topology.topology_list = TopologyList.SHUNT_STUB_RESONATORS
+        design.topology.equal_stub_widths = False
+        assert design.topology.center_z0_impedance == "75"
+        design.topology.center_z0_impedance = "55"
+        assert design.topology.center_z0_impedance == "55"
+        assert design.topology.circuit_response().splitlines() == read_resource_file(
+            "center_z0_impedance.ckt", "Distributed"
+        )
+
+    def test_equal_width_conductors(self):
+        design = ansys.aedt.core.filtersolutions.DistributedDesign(config["desktopVersion"])
+
+        with pytest.raises(RuntimeError) as info:
+            design.topology.equal_width_conductors = True
+        assert info.value.args[0] == "The Equal Width property is not supported for the selected topology"
+        design.attributes.filter_class = FilterClass.BAND_PASS
+        design.topology.topology_list = TopologyList.PARALLEL_EDGE_COUPLED
+        assert design.topology.equal_width_conductors
+        design.topology.equal_width_conductors = False
+        assert design.topology.equal_width_conductors is False
+        assert design.topology.circuit_response().splitlines() == read_resource_file(
+            "equal_width_conductors.ckt", "Distributed"
+        )
+
+    def test_tapped(self):
+        design = ansys.aedt.core.filtersolutions.DistributedDesign(config["desktopVersion"])
+
+        with pytest.raises(RuntimeError) as info:
+            design.topology.tapped = True
+        assert info.value.args[0] == "The Tapped property is not supported for the selected topology"
+        design.attributes.filter_class = FilterClass.BAND_PASS
+        design.topology.topology_list = TopologyList.PARALLEL_EDGE_COUPLED
+        assert design.topology.tapped
+        design.topology.tapped = False
+        assert design.topology.tapped is False
+        assert design.topology.circuit_response().splitlines() == read_resource_file("tapped.ckt", "Distributed")
+
+    def test_pinned(self):
+        design = ansys.aedt.core.filtersolutions.DistributedDesign(config["desktopVersion"])
+
+        with pytest.raises(RuntimeError) as info:
+            design.topology.pinned = True
+        assert info.value.args[0] == "The Pinned property is not supported for the selected topology"
+        design.attributes.filter_class = FilterClass.BAND_PASS
+        design.topology.topology_list = TopologyList.PARALLEL_EDGE_COUPLED
+        assert design.topology.pinned is False
+        design.topology.pinned = True
+        assert design.topology.pinned
+        assert design.topology.equal_width_conductors is False
```

Comment:
Should it be enabled to evaluate the circuit if we want to test the pinned option?